app/src/main/java/com/example/moodtrackerproject/ui/mood/list/MoodFragment.kt

```diff
@@ -39,7 +41,19 @@ class MoodFragment : BaseFragment<MoodViewModel, FragmentMoodBinding, MoodProps>
         binding?.run {
             moodsAdapter.submitList(props.listOfMoods)
             moodInclude.addMoodBtn.click(props.addNewMood)
-            stressInclude.root.click(props.openStressTestScreen)
+            stressInclude.root.click {
+                DataBaseRepository.stressPoints = 0
```

Comment:
No, no, no...
1) no repository access in UI - `DataBaseRepository.stressPoints = 0`
2) no setState in UI - ` Store.setState(Store.appState.stressTestState.copy(testType = 0))`
3) props don't have setters - `props.setTestType(0)` - props are filled only in ViewModel
4) for routing operations you have Action in props - ` props.openStressTestScreen()`